docs/notebooks/index.md

```diff
@@ -51,9 +51,10 @@ The Digital Assets contract family implements a basic digital asset for bitmap i
 ### Inference
 
 
-The Inference contract family provides contracts for creating a confidentiality preserving policy-wrapper around the usage of a machine learning (ML) model. At its core, the implementation uses a token contract to  specify and enforce policies to be followed while using the ML model for inferecing operations. 
+The Inference contract family provides contracts for creating a confidentiality
+preserving policy-wrapper around the usage of a machine learning (ML) model.
```

Comment:
What is a policy-wrapper?

Reply:
Thank you, have clarified about both policy-wrapper and token object : "The policy-wrapper specifies and checks the
policies to be followed while using the ML model for inferencing
operations, and is implemented as a PDO contract that we refer to as the Token Object."